gatsby-template/src/templates/blog-post.css

```diff
@@ -1,3 +1,9 @@
+.blog h1 a {
+  text-decoration: none;
+  font-size: 48px;
+  color: lavender;
```

Comment:
```suggestion
  color: lavender;
  margin-left: -40px;
```

to offset it 
<img width="637" alt="image" src="https://user-images.githubusercontent.com/3254314/58746823-3cd36500-8439-11e9-93d0-1c4a5c84ae6a.png"> ?